### ISSUE TITLE
Compatibility with 2017.2

### ISSUE THREAD

**Issue by pirocks** [intellij-dlanguage/intellij-dlanguage#210]
Currently we are not compatible.

**Comment by SingingBush** [comment]
Well the good news is that the develop branch is fine to build and test on 2017.2 https://travis-ci.org/intellij-dlanguage/intellij-dlanguage/builds/255336148

I also ran it with 2017.2 locally and everything seemed fine (although didn't do much testing) nothing seemed broken.

So I've just adjusted the `plugin.xml` in develop so it can support any 2017.1.* and 2017.2.* versions of Intellij.
I'll merge develop into master again, hopefully the deployment will work this time

**Comment by SingingBush** [comment]
after many tweaks and waiting for travis I've finally got the new travis-ci Build Stages working. This seems better than the way we previously attempted deployments.
![travis-build-stages](https://user-images.githubusercontent.com/870567/28394451-042a8b6a-6ce5-11e7-9302-41b73ba66665.png)
It didn't seem to notice when I tagged 1.13 though so from now on master will deploy EAP builds and there's a release branch that is used for deploying stable builds. This seems to be working well:

https://travis-ci.org/intellij-dlanguage/intellij-dlanguage/builds/255470185

https://plugins.jetbrains.com/plugin/8115-d-language
